crafts/css/styles.css

```diff
@@ -0,0 +1,73 @@
+body {
```

Comment:
I was thinking of such a directory structure.

```
| -- css
| -- index.html
| -- pages
    | -- crafts
        | -- css
        | -- js
        | -- crafts.html
```

I think it's good to have a structure like this, we can have folders called `alumni/` under `pages/` as and when required.

Reply:
alright, I will do this in the next push.

Reply:
Is it okay for now to approve I have implemented other changes to the sites too